batch_seqr_loader/batch_workflow.py

```diff
@@ -644,8 +688,13 @@ def _add_merge_gvcfs_job(
 
     j.command(
         f"""set -e
-    java -Xms{mem_gb - 1}g -jar /usr/picard/picard.jar \
+
+    (while true; do df -h; pwd; free -m; sleep 300; done) &
```

Comment:
Did you mean to leave this here? (Fine by me, just checking :) )

(Also below.)

Reply:
Yeah, wanted to monitor the disk usage for a while to have a better idea and optimize it more.